### ISSUE TITLE
Can't resolve velocity on create-react-app v2

### ISSUE THREAD

**Issue by mustafahlvc** [google-fabric/velocity-react#244]
Hi,

I ve tested velocity-react on create-react-app@next ,
create-react-app 2.0.0 will be stable soon:
https://github.com/facebook/create-react-app/issues/5103#issue-364148526 

I m getting **Can't resolve 'velocity'** error.

I m not sure but it could be related with webpack and updating velocity-animate version could solve this issue? https://github.com/julianshapiro/velocity/issues/743#issuecomment-372103588

Could you please checkout the issue?

**Comment by finneganh** [comment]
Where are you seeing that? What version?

The latest version of `velocity-react` has `require('velocity-animate')`. See `velocity-animate-shim.js`

**Comment by mustafahlvc** [comment]
I'm getting the error using "react-scripts": "2.0.0" and importing as below: 

```
import {VelocityComponent} from 'velocity-react';
import 'velocity-animate';
import 'velocity-animate/velocity.ui';
```
Thanks in advance.

**Comment by finneganh** [comment]
What version of `velocity-react`? Is there a stack trace for the resolve error?

**Comment by mustafahlvc** [comment]
"velocity-react": "1.4.1"

![image](https://user-images.githubusercontent.com/4725291/46169567-2ff7e380-c2a4-11e8-854b-d4c4cbfb8ac5.png)


When i add   "velocity-animate": "^2.0.5" manually it works but face off  the ui pack issue due to multiple velocity-animate package.

> If, even with the above statements, you see errors like Velocity: First argument (transition.shrinkIn) was not a property map, a known action, or a registered redirect. Aborting. it is likely that there are 2 copies of velocity-animate in your node_modules. Use npm dedupe to collapse them down to one.

**Comment by mustafahlvc** [comment]
Hi,

The issue solved with the release  "react-scripts": "2.0.4".

Thanks.
